### PR TITLE
fix: Fixed model_proto serialized dump in Debug

### PR DESCRIPTION
**Pull request by ankitm3k**
### Description
This PR fixes below issue for Debug build -

image

### CODE REVIEW

**Comment by sfatimar** [comment]
lgtm

**Comment by preetha-intel** [comment]
Can we remove this model dump. Its redundant actually. There is a model dump happening at 
https://github.com/intel/onnxruntime/blob/524d766f95dd717aab0cdcf1651ac9e0e3651572/onnxruntime/core/providers/openvino/backend_manager.cc#L324 

and 

https://github.com/intel/onnxruntime/blob/524d766f95dd717aab0cdcf1651ac9e0e3651572/onnxruntime/core/providers/openvino/backend_manager.cc#L334

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc [hunk 1]
```diff
@@ -52,7 +52,7 @@ BasicBackend::BasicBackend(std::unique_ptr<ONNX_NAMESPACE::ModelProto>& model_pr
   if (IsDebugEnabled()) {
     std::string file_name = subgraph_context.subgraph_name + "_static.onnx";
     std::fstream outfile(file_name, std::ios::out | std::ios::trunc | std::ios::binary);
-    model_proto.SerializeToOstream(outfile);
+    model_proto->SerializeToOstream(outfile);
   }
 #endif
```
Comment (preetha-intel) [inline onnxruntime/core/providers/openvino/backends/basic_backend.cc:58]: Remove this whole block from 52 to 57

**Comment by saurabhkale17** [comment]
> Can we remove this model dump. Its redundant actually. There is a model dump happening at
> 
> https://github.com/intel/onnxruntime/blob/524d766f95dd717aab0cdcf1651ac9e0e3651572/onnxruntime/core/providers/openvino/backend_manager.cc#L324
> 
> and
> 
> https://github.com/intel/onnxruntime/blob/524d766f95dd717aab0cdcf1651ac9e0e3651572/onnxruntime/core/providers/openvino/backend_manager.cc#L334



Model dumping on line 334 can be removed but I won't suggest the removal of model dumping on 324.
It (324) enables us to dump the qdq stripped model just by setting the debug flag, not requiring the debug build. (Used by many teams for WIN-AI)

**Comment by sfatimar** [comment]
Please raise this PR for ovep-develop-lnl-1.2 branch too

**Pull request merged**